nodebb

```diff
@@ -2,36 +2,34 @@
 
 'use strict';
 
-var cproc;
-var args;
-var fs;
-var path;
-var request;
-var semver;
-var prompt;
-var async;
+var fs = require('fs');
+var path = require('path');
+var cproc = require('child_process');
 
+// check to make sure dependencies are installed
 try {
-	require('colors');
-	cproc = require('child_process');
-	args = require('minimist')(process.argv.slice(2));
-	fs = require('fs');
-	path = require('path');
-	request = require('request');
-	semver = require('semver');
-	prompt = require('prompt');
-	async = require('async');
+	fs.readFileSync(path.join(__dirname, 'node_modules/async/package.json'));
```

Comment:
Right, but I think that any call to `npm install` will install all packages, even if it's only called to install a certain plugin. So unless packages were deleted manually they're either all the or none are there, I think.